docs/src/data/mockServices.tsx

```diff
@@ -0,0 +1,78 @@
+let signUpPassword = '';
+const verifiedUsers = new Map();
+const FIXED_USERNAME = 'test';
+const FIXED_PASSWORD = 'password';
+
+export const mockServices = {
+  async getAmplifyConfig() {
+    return {};
+  },
+  async handleSignUp(formData) {
+    signUpPassword = formData.password;
+    return {};
+  },
+  async checkVerifiedContact() {
+    return { verified: {}, unverified: {} };
+  },
+  async handleSignIn({
+    username,
+    password,
+  }: {
+    username: string;
+    password: string;
+  }) {
+    if (
+      verifiedUsers.get(username).password === password ||
+      (username === FIXED_USERNAME && password === FIXED_PASSWORD)
```

Comment:
Can we move this in to a meaningfully name util at the top of this file?

Reply:
I think this is the remaining nit: util function for checking fixed username + pw